docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_monitoring/_index.md

```diff
@@ -0,0 +1,235 @@
+---
+type: docs
+title: "Azure Monitor"
```

Comment:
We need a more descriptive name for the scenario. Azure Monitor is the primary component being showcased but a title that captures more of the Operations Management aspect of the entire solution would be nice. 

@lanicolas interested in your feedback here also.

Reply:
how about Monitoring and alerting for Azure Arc-enabled servers? I would not focuse it just on the Azure Monitor product as it expands more

Reply:
Considering that none of the scenarios have **Azure Arc-enabled servers** on the title:
- Inventory Tagging
- Monitoring Agent Extension
- Custom Script Extension
- Azure Policy
- Key Vault Integration
- Microsoft Defender for Cloud
- Azure Sentinel
- Update Management
- Azure Automanage
- Managed Identity
- Azure Security Baseline

Therefore, I would suggest something like:
- E2E Operating System monitoring
- Alerts, Insights and Workbooks
- Monitoring, Alerting and Visualization

Reply:
I like Monitoring, Alerting, and Visualization

Reply:
Thanks @dkirby-ms! I've already changed the title and I've completed all the requested changes. Please, do let me know if I need to updated anything else. Thanks for your time on reviewing it.